src/components/custom/signup.js

```diff
@@ -91,6 +91,7 @@ class Signup extends Component {
         this.setState({
             [name]: value,
         })
+        this.handleValidation(value)
```

Comment:
can you help to make a todo to change this to formik 🙏

Reply:
Will add a trello card to the backlog